src/xunit.runner.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file moved to `targets/xunit.runner.json`--can you update the `.sln` to point to that location?